src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/compile.py

```diff
@@ -78,7 +78,7 @@ def openvino_compile_cached_model(cached_model_path, options, *example_inputs):
 
     return compiled_model
 
-def openvino_compile(gm: GraphModule, *args, model_hash_str: str = None, options=None):
+def openvino_compile(gm: GraphModule, *args, model_hash_str: str = None, options=None, executorch=False):
     core = Core()
 
     device = _get_device(options)
```

Comment:
for `ExecuTorch`, will you cache `ov::Model` or `ov::CompiledModel`?

Reply:
Since the generated pte file supposed to include the compiled model already, we are still discussing whether we need model caching for executorch. For the initial implementation, we can disable caching for executorch. I will update accordingly..